src/rust/ensogl/lib/components/src/selector/bounds.rs

```diff
@@ -56,7 +56,7 @@ impl From<(f32,f32)> for Bounds {
 /// Frp utility method to normalise the given value to the given Bounds.
 ///
 /// Example usage:
-/// ```ignore
+/// ```text,.ignore
```

Comment:
This is what rustdoc suggested but actually it does not make sense. Blocks parsing as Rust should be `ignore`, otherwise `text`. Using both at once useless.